common/lanelet2_extension/package.xml

```diff
@@ -8,10 +8,16 @@
 
   <license>Apach 2</license>
 
-  <buildtool_depend condition="$ROS_VERSION == 1" >catkin</buildtool_depend>
+  <buildtool_depend condition="$ROS_VERSION != 2" >catkin</buildtool_depend>
+  <buildtool_depend condition="$ROS_VERSION == 2">ament_cmake</buildtool_depend>
+  
+  <build_depend condition="$ROS_VERSION == 2">ament_cmake_auto</build_depend>
+  
+  <build_depend>carma_cmake_common</build_depend>
 
-  <depend condition="$ROS_VERSION == 1 and $LANELET2_EXTENSION_LOGGER_TYPE == 0" >roscpp</depend>
-  <depend condition="$ROS_VERSION == 2 and $LANELET2_EXTENSION_LOGGER_TYPE == 0" >rclcpp</depend>
+
+  <depend condition="$ROS_VERSION == 1" >roscpp</depend>
```

Comment:
Looks like this comparison is being used interchangeably between !=2 and ==1, is there a reason for it?

Reply:
Yeah, this package is used in carma-streets which has catkin but not ROS installed. So I only want roscpp to link when ROS 1 is available but catkin should always link except when ROS2 is available.